docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/build-aspnet-core-applications-linux-containers-aks-kubernetes.md

```diff
@@ -29,37 +29,37 @@ This example uses a simple project that's based based on a Visual Studio Web API
 
 **Figure 4-36**. Creating ASP.NET Core Application
 
-To create the sample project, you have to select **File** > **New** > **Project** on Visual Studio. Then you'll see a list of templates for several types of projects, where you have to look for **Web** > **.NET Core** on the left panel. For this example select **ASP.NET Core Web Application**.
+To create the sample project in Visual Studio, select **File** > **New** > **Project**, select the **Web** project types in the left pane, followed by **ASP.NET Core Web Application**.
 
-In the next dialog ensure that you have selected .NET Core and ASP.NET Core 2.1 as the target framework in the top pulldowns, as shown in figure 4-37, and then select the API option, to create an ASP.NET Core Web API application.
+Visual Studio lists templates for web projects. For our example, you select **Web API,** to create an ASP.NET Web API Application.
```

Comment:
```suggestion
Visual Studio lists templates for web projects. For our example, select **API** to create an ASP.NET Web API Application.
```
At least in my VS version, the text is "API" instead of "Web API"